File: 3753/3753.c
int totalWavinessOfNumbersInRangeI(struct TreeNode* root) {
    if (!root) return 0;
    int l = totalWavinessOfNumbersInRangeI(root->left);
    int r = totalWavinessOfNumbersInRangeI(root->right);
    return 1 + (l > r ? l : r);
}

File: 3753/3753.ts
function totalWavinessOfNumbersInRangeI(root: TreeNode | null): number {
    if (!root) return 0;
    return 1 + Math.max(totalWavinessOfNumbersInRangeI(root.left), totalWavinessOfNumbersInRangeI(root.right));
}